packages/dds/tree/src/shared-tree-core/sharedTreeCore.ts

```diff
@@ -115,7 +115,7 @@ export class SharedTreeCore<
 	/**
 	 * Provides events that indexes can subscribe to
 	 */
-	private readonly indexEventEmitter = createEmitter<IndexEvents<TChange>>();
+	public readonly indexEventEmitter = createEmitter<IndexEvents<TChange>>();
```

Comment:
making this public would allow users of this class to trigger events (not just subscribe to them). I'm not sure what the usecase here is, but the general pattern is we don't expose emitters other than as ISubscribable.

Also these events are for the tree as a whole and not the specific view/branch of the tree you are interacting with. Can your usecase be solved with the new "afterBatch" event accessed via `ISharedTreeBranch.events`?

Reply:
This change is obsolete. Leaked accidentally from earlier versions of a quick & dirty prototype which required `newLocalState` before `afterBatch` was available. I confirm that current version uses `afterBatch` to detect completion.